### ISSUE TITLE
Gravity bomb range too far on XT-002

### ISSUE THREAD

**Issue by cjwyatt** [dalaranwow/dalaran-wow#5235]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Players 15 yards from players with gravity bomb are being pulled in and taking damage.
**Expected behaviour**: Tell us what should happen instead
Gravity bomb should only have a 12 yard range.
**Steps to reproduce the problem**:

1. Fight XT-002
2. Stand 15 yards away from a player with gravity bomb
3. Still get pulled in

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @cjwyatt.
I've look in to the spell Gravity Bomb and you can see here:
https://wowgaming.altervista.org/aowow/?spell=63025 10m version
http://www.wowhead.com/spell=63025/gravity-bomb 10m version

**Effect 2 | Pull Towards: (140)Value: 140Radius: 20 yards**



https://wowgaming.altervista.org/aowow/?spell=64233 25m version
http://www.wowhead.com/spell=64233/gravity-bomb 25m version

**Effect 2 | Pull Towards: (140)Value: 140Radius: 20 yards**

The 12 yards is the radius of the black voidzone in which the players take damage (12 yards from the center of the zone).

If there is more relevent information I'll reopen the report
